src/DevChatter.DevStreams.Core/Data/ICrudRepository.cs

```diff
@@ -11,6 +11,7 @@ public interface ICrudRepository
         Task<List<T>> GetAll<T>();
         Task<List<T>> GetAll<T>(string filter, object args);
         Task<List<T>> GetAll<T>(string filter, string orderBy, object args);
+        Task<List<Channel>> GetAllChannelInfo();
```

Comment:
`ICrudRepository` should be for generic methods and types only, so we won't want `Channel` to be known by it.